lib/api/controllers/adminController.js

```diff
@@ -51,6 +53,107 @@ function AdminController(kuzzle) {
   };
 
   /**
+   * Get the collection validation specifications
+   * @param {RequestObject} requestObject
+   * @returns {Promise}
+   */
+  this.getSpecifications = requestObject => {
```

Comment:
Doesn't this break the new eslint policy about unnamed function?
If not, can we easily retrieve the corresponding code from a coredump if this function causes it (or is in the stack trace), or is it listed as an "anonymous function"?

@stafyniaksacha > what do you think about it?